prime-angular-frontend/src/app/modules/enrolment/enrolment.routes.ts

```diff
@@ -40,6 +42,16 @@ export class EnrolmentRoutes {
   public static ENROLMENT = 'enrolment';
   public static ABSENCE_MANAGEMENT = 'absence-management';
 
+  // Step display text
+  public static SETP_CONTACT = 'Contact';
+  public static STEP_CARE_SETTING = 'Care Setting';
+  public static STEP_COLLEGE = 'College /<br/>Job Info';
+  public static STEP_SELF_DECLARATION = 'Self-Declaration';
+  public static STEP_REVIEW = 'Review';
+  public static STEP_TOA = 'Term of Access';
```

Comment:
Should be "Terms of Access"

![image](https://user-images.githubusercontent.com/7586665/222460557-d95bc09a-9441-4ccf-9c7e-4bc1407a342c.png)